src/Scripting/Core/CommandLine/CommandLineHostObject.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Scripting.Hosting
+{
+    public sealed class CommandLineHostObject
+    {
+        private readonly TextWriter _outputWriter;
+        private ObjectFormattingOptions _formattingOptions;
```

Comment:
These should be settable -- I'm thinking adding PrintSettings property that changes the formatting options. I need to review all properties on this object, unify them with InteractiveHost and document them.